src/Modules/Family/AddressComponent.js

```diff
@@ -1,59 +1,105 @@
-import React, { Fragment, forwardRef } from 'react';
-import StateDropdownComponent from './StateDropdownComponent';
+import React, { Fragment, forwardRef } from "react";
+import StateDropdownComponent from "./StateDropdownComponent";
+import PlacesAutocomplete from "react-places-autocomplete";
 
-const AddressComponent = forwardRef(({ register, errors }, ref) => (
-  <Fragment>
-    <h2>Where you live</h2>
-    <div className="form-group">
-      <label htmlFor="address_line_1">Street Address</label>
-      <input
-        type="text"
-        className="form-control"
-        name="address_line_1"
-        id="address_line_1"
-        ref={register({ required: true })}
-      />
-      {errors.address_line_1 && <span className="text-danger">This field is required</span>}
-    </div>
+const AddressComponent = forwardRef(({ register, errors }, ref) => {
+  const [addressLine1, setAddressLine1] = React.useState("");
```

Comment:
@seanlawrenz my understanding of the state management is limited, but does this approach build off of the work you have done with Redux?

Reply:
This is component level state management. This should be fine to use here.